rskj-core/src/main/java/co/rsk/core/RskFactory.java

```diff
@@ -415,7 +415,13 @@ public TransactionGateway getTransactionGateway(
     public BuildInfo getBuildInfo(ResourceLoader resourceLoader) throws IOException {
         Properties props = new Properties();
         Resource buldInfoFile = resourceLoader.getResource("classpath:build-info.properties");
-        props.load(buldInfoFile.getInputStream());
+        try {
+            props.load(buldInfoFile.getInputStream());
+        } catch (IOException ioe) {
+            logger.warn("build-info.properties file missing from classpath");
```

Comment:
Nice logging pattern.

Reply:
Honestly I'm not quite sure this is the better way. I think you can attach different pattern layouts through Logback configuration according the [top] level enabled but I didn't investigate further. We have a deep debt on how we are handling logging, we can add this as an improvement path when we face it